src/js/multiDomainFirstParties.js

```diff
@@ -126,7 +148,6 @@ var multiDomainFirstPartiesArray = [
     "aliloan.com",
     "alipay-cloud.com",
     "alipay.cn",
-    "alipay.com",
```

Comment:
Does `alipay.com` not belong to Alipay?

Reply:
No. The domain is already present in L124(prev)/146(now). I think there should not be duplicated domains on the list? BTW, the check script should also cover this (dups)?

Reply:
Ah, I see, a copy-paste typo in my part. Thanks!

Reply:
I added a check to prevent duplicates in the future: 6d4dbcfa8417b082277e3b9c7b88f67f97807d7e